### PR TITLE
DOC: update link to glasser supplementary info; convert to footbib

### PR DESCRIPTION
**Pull request by drammock**
fixes a 404 link, and converts the docstring to use `footcite` while we're at it.

### CODE REVIEW

File: doc/references.bib [hunk 1]
```diff
@@ -376,13 +376,23 @@ @article{FishburnEtAl2019
   publisher={Elsevier}
 }
 
-@article{GlasserEtAl2016,
+@article{GlasserEtAl2016a,
```
Comment (larsoner) [inline doc/references.bib:379]: instead of a/b, maybe better to leave this one and have `supp` for the other one?

**Comment by larsoner** [comment]
CircleCI is broken, @GuillaumeFavelier can you look? It has to do with `pyvistaqt`. Locally when I switched to using `pyvista` master and `pyvistaqt` there were some tests that were not happy...

In the meantime, let's pin `pyvista<0.25` (this PR is fine) to avoid this problem @drammock

**Comment by drammock** [comment]
hmm... a bunch of failures in `test_brain` and `tst_3d` in the azure 3.7 conda build, see [here](https://dev.azure.com/mne-tools/mne-python/_build/results?buildId=7477&view=logs&j=a846d25a-e32c-5640-1b53-e815fab94407&t=f8799db5-b972-5f58-516d-bc88b8357f99&l=356).

On the other hand, [the docstring changes look good](https://20411-1301584-gh.circle-artifacts.com/0/dev/generated/mne.datasets.fetch_hcp_mmp_parcellation.html)

**Comment by GuillaumeFavelier** [comment]
I see that pyvista has been pinned to 0.24 in requiremenrs.txt but not in environment.yml. That could explain the failures on conda

**Pull request merged**